prod-sample/sample-modules/unhealthy-frank/0.0.0/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "unhealthy-frank",
```

Comment:
I wonder if we can reuse any of the other franks rather than creating a brand new one? If we deploy the correct version of the root module that doesn't provide any externals, I think `cultured-frankie@0.0.1` should be enough?

Reply:
It was easier to add a new module that only had one version with an issue, than to manage an existing module with one working version and one broken one

Reply:
I am just thinking that it is more work to maintain another frank but I guess now we removed the package-locks from them we shouldn't need to change them too much.

Is it achievable with the franks that we already got? if so maybe double check with the team to see what's their preference

Reply:
maintenance aside, building another module would also be expensive and increase integration tests build time

Reply:
An additional version of an existing module would have the same downside.